eventkit_cloud/tasks/export_tasks.py

```diff
@@ -1265,11 +1266,12 @@ def cancel_synchronous_task_chain(data_provider_task_uid=None):
         if TaskStates[export_task.status] == TaskStates.PENDING.value:
             export_task.status = TaskStates.CANCELED.value
             export_task.save()
+            queue_group = os.getenv("CELERY_GROUP_NAME", export_task.worker)
             kill_task.apply_async(
                 kwargs={"task_pid": export_task.pid, "celery_uid": export_task.celery_uid},
-                queue="{0}.cancel".format(export_task.worker),
```

Comment:
This should remain as the worker (i.e. the hostname of specific container), otherwise there is no way to send a message to a specific machine to cancel a specific process.